pkg/mimirpb/mimir.proto

```diff
@@ -48,7 +48,6 @@ message LabelPair {
 }
 
 message Sample {
-  // Fields order MUST match promql.Point so that we can cast types between them.
```

Comment:
Is this comment no longer true?

Reply:
There is no longer Point, only FPoint and HPoint, so we don't just cast directly anymore, we write it out properly.

Reply:
Ah of course, sorry 🙂